test2code/src/test/kotlin/com/epam/drill/test2code/coverage/BufferTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.epam.drill.test2code.coverage
+
+import com.epam.drill.common.agent.*
+import com.epam.drill.jacoco.*
+import com.nhaarman.mockitokotlin2.*
+import kotlinx.coroutines.*
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Assertions.*
+import java.util.*
+import java.util.concurrent.atomic.*
+import kotlin.random.Random
+
+class BufferTest {
+
+    // General services and vars at test execution
+    private lateinit var sender: Sender
+    private lateinit var id: String
+    private lateinit var coverageSender: CoverageSender
+    private lateinit var inMemoryBuffer: InMemoryBuffer
+    private lateinit var coverageTransport: CoverageTransport
+
+    @BeforeEach
+    fun setUp() {
+        id = UUID.randomUUID().toString()
```

Comment:
Introducing `id` and using UUID is an unnecessary complication.
See comment below to `verify(sender, times(1)).send(eq(id), any())` for more info

Reply:
done